coalib/parsing/FilterHelper.py

```diff
@@ -0,0 +1,38 @@
+from coalib.output.printers.LogPrinter import LogPrinter
+
+from coalib.parsing.filters.LanguageFilter import LanguageFilter
+from coalib.parsing.filters.CanDetectFilter import CanDetectFilter
+from coalib.parsing.filters.CanFixFilter import CanFixFilter
+
+
+class FilterHelper:
+    all_filters = {'language': LanguageFilter,
+                   'can_detect': CanDetectFilter,
+                   'can_fix': CanFixFilter}
+
+    @classmethod
+    def get_all_filters_str(cls, sep=', '):
+        return sep.join(sorted(cls.all_filters))
```

Comment:
Its also being used in `FilterHelper.get_help_message` function. So, I think it makes sense not to repeat it at bother places.